test/test.js

```diff
@@ -94,4 +99,25 @@ describe('cypress-terminal-report', () => {
       });
     });
   }).timeout(60000);
+
+  it('Logs cy.requests', async () => {
+    await new Promise(resolve => {
+      exec(commandBase('printLogs=always') + 'requests.spec.js', (error, stdout, stderr) => {
+        expect(stdout).to.contain(
+          'cy:command ✔  request\tGET https://jsonplaceholder.cypress.io/todos/1\n\t\t\t\tStatus: 200 \n\t\t\t\tResponse: \n\t\t\t\t{\n\t\t\t\t  "userId": 1,\n\t\t\t\t  "id": 1,\n\t\t\t\t  "title": "delectus aut autem",\n\t\t\t\t  "completed": false\n\t\t\t\t}\n\n\n\n'
```

Comment:
should I use `json-server` https://www.npmjs.com/package/json-server?

Reply:
For what would we need that?

Reply:
I hope not. This test relies on `https://jsonplaceholder.cypress.io`. It will fail if they response change.

Reply:
Its fine for now. We can change it in the future if it gets in the way